docs/KnownLimitations.md

```diff
@@ -0,0 +1,47 @@
+# Known Limitations
+
+List where libical doesn't fully support all the features described in the RFCs.
+
+Contributions to help us fix these limitations are welcome.
+
+## RFC 5545
+
+### Recurrence Rule section 3.3.10
+
+* BYSETPOS is only supported for MONTHLY and YEARLY frequencies (<https://github.com/libical/libical/issues/795>)
+
+* When used as a limit (rather than for expansion) negative values are not supported
+  for BYMONTHDAY, BYYEARDAY. possibly others (<https://github.com/libical/libical/issues/791>)
+
+* FREQ=YEARLY, BYYEARDAY can't be combined with BYWEEKNO, BYMONTH or BYMONTHDAY
+
+     ref: <https://github.com/libical/libical/blob/cfd401b9d043214395888de1d9daf52263e3245b/src/libical/icalrecur.c#L2882>
+
+     ```c
```

Comment:
Don't think we need to add the code snippet here. By adding the link, users are free to look it up if they're interested.